net/upnp/src/opnsense/mvc/app/models/OPNsense/UPnP/Menu/Menu.xml

```diff
@@ -1,10 +1,10 @@
 <menu>
     <Services>
-        <UPnP VisibleName="Universal Plug and Play" cssClass="fa fa-plug fa-fw">
-            <Settings url="/services_upnp.php">
+        <UPnP VisibleName="UPnP IGD &amp; PCP" cssClass="fa fa-plug fa-fw">
```

Comment:
I'm not really a fan of the ampersand and then stuffing it into XML which will force encoding quirks for no apparent reason

Reply:
> Also used in Dnsmasq DNS & DHCP

I'm not entirely fond of that either, but

> Currently, there is only an undecoded display of user-privileges

Yes, as expected. Let's go with your suggestion but I trade you for a core ticket for the issue?